### PR TITLE
Social accounts: disable disconnect button if the account can't be disconnected

### PR DESCRIPTION
**Pull request by stsewd**
Closes https://github.com/readthedocs/readthedocs-corporate/issues/1019

Needs https://github.com/readthedocs/readthedocs.org/pull/11503

Screenshot

### CODE REVIEW

File: readthedocsext/theme/templates/socialaccount/partials/social_account_list.html [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 {% load i18n %}
 {% load gravatar %}
+{% load can_be_disconnected from readthedocs.socialaccounts %}
```
Comment (stsewd) [inline readthedocsext/theme/templates/socialaccount/partials/social_account_list.html:5]: Namespaces are great!

Comment (agjohnson) [reply readthedocsext/theme/templates/socialaccount/partials/social_account_list.html:5]: I didn't know about this! I would like to use this as much as possible. Wildcard imports in templates are very confusing and I don't like struggling to guess where template tags/filters are coming from.

**Review by agjohnson — APPROVED** [review]
Looks great!

File: readthedocsext/theme/templates/socialaccount/partials/social_account_list.html (hunk 1 above)
Comment (stsewd) [reply readthedocsext/theme/templates/socialaccount/partials/social_account_list.html:5]: Same, it's nice going back to the Django docs and find good things like this :D

**Pull request merged**